android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -105,6 +107,7 @@ class DVCClient private constructor(
         } else {
             User.builder().withUserParam(user, context).build()
         }
+        latestIdentifiedUser = updatedUser
```

Comment:
you should also set this in the resetUser method

Reply:
good catch - fixed!